src/sqlancer/Main.java

```diff
@@ -449,9 +450,11 @@ public void run() throws Exception {
                 if (options.reduceAST() && !options.useReducer()) {
                     throw new AssertionError("To reduce AST, use-reducer option must be enabled first");
                 }
-                if (reproducer != null && options.useReducer()) {
-                    System.out.println("EXPERIMENTAL: Trying to reduce queries using a simple reducer.");
-                    // System.out.println("Reduced query will be output to stdout but not logs.");
+                if (options.useReducer()) {
+                    if (reproducer == null) {
+                        logger.getQueryPlanFileWriter().write("current oracle do not support experimental reducer.");
```

Comment:
I think this should be "does not support" since the message is about a specific oracle. Other than that, the PR looks good to me.